README.md

```diff
@@ -1,25 +1,24 @@
-# SEO: A plugin for Pelican :fr:
+# SEO Plugin for Pelican :fr:
 
 [![Build Status](https://img.shields.io/github/workflow/status/pelican-plugins/seo/build)](https://github.com/pelican-plugins/seo/actions)
 
-This plugin helps you improve your Pelican site SEO (Search Engine Optimization) to reach the top positions on search engines. To see what can do SEO for you, go directly to [Usage](#usage) section.
+This plugin helps you improve your Pelican site SEO (Search Engine Optimization) to reach the top positions on search engines. To see what this SEO plugin can do for you, visit the [Usage](#usage) section.
 
 | Author          | GitHub                            |
 | :-------------: | :-------------------------------: |
 | Maëva Brunelles | https://github.com/MaevaBrunelles |
 
-## Why do you need SEO
+## Why
+
+You might want to use this plugin if:
 
-If :
-* you need some help to improve the SEO of your Pelican site
 * you are not familiar with SEO
-* you want a simple site with all SEO basis
+* you need help improving the SEO of your Pelican site
 
-This plugin is for you !
```

Comment:
Same thougth as for the title. I added that line to have a more user-frienfly README, not only for tech people.

Reply:
Perhaps format it this way, in order to create continuity:

    … then this plugin is for you!

Reply:
That would be nice!